src/qtism/data/storage/xml/marshalling/PositionObjectInteractionMarshaller.php

```diff
@@ -89,7 +98,15 @@ protected function unmarshall(DOMElement $element)
                     $component->setMaxChoices($maxChoices);
                 }
 
-                if (Version::compare($version, '2.1.0', '>=') === true && ($minChoices = $this->getDOMElementAttributeAs($element, 'minChoices', 'integer')) !== null) {
+                if (Version::compare(
+                        $version,
+                        '2.1.0',
+                        '>='
+                    ) === true && ($minChoices = $this->getDOMElementAttributeAs(
+                        $element,
+                        'minChoices',
+                        'integer'
+                    )) !== null) {
```

Comment:
could you provide here proper multiline if statement as suggested earlier or revert that place